src/cdk/schematics/index.ts

```diff
@@ -9,3 +9,10 @@
 export * from './utils';
 export * from './ng-update/public-api';
 export * from './update-tool/public-api';
+
+// Re-export parse5 from the CDK. Material schematics code cannot simply import
+// "parse5" because it could result in a different version. As long as we import
+// it from within the CDK, it will always be the correct version that is specified
+// in the CDK "package.json" as optional dependency.
+import * as parse5 from 'parse5';
+export {parse5};
```

Comment:
Yeah, one other solution would be to import parse5 deeply, but it would involve first checking `node_modules/@angular/cdk/node_modules/parse5` and then `node_modules/parse5` as fallback.

I think the risk is low. We marked `cdk/schematics` as private-only and there is already plenty of other stuff exported 🤷‍♂